dockerfiles/backfill_events/startup_script.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 
+ENDING_BLOCK_NUMBER=10875761
```

Comment:
I'm wondering if we might want to remove the `ENDING_BLOCK_NUMER` var from this file entirely and just hard code `10875761` as the value passed to `-e` ... not sure if an env var in the deploy could end up overriding the value we're assigning here

Reply:
oh yeah, good point - like if we have an ENDING_BLOCK_NUMBER set in the environment already, it could overwrite this value. i'll update shortly.